packages/wonder-blocks-form/src/components/checkbox-core.tsx

```diff
@@ -50,13 +68,31 @@ export default class CheckboxCore extends React.Component<ChoiceCoreProps> {
             "data-test-id": testId,
         } as const;
 
+        const checkboxIcon = (
+            <Icon
+                color={disabled ? offBlack32 : white}
+                icon={checked ? checkPath : indeterminatePath}
+                size="small"
+                style={sharedStyles.checkboxIcon}
+            />
+        );
+
+        const ariaChecked = mapCheckedToAriaChecked(checked);
+
+        // TODO (nicolemejia) - We're not using `.indeterminate` on
+        // `StyledInput` because `addStyle()` doesn't support ref forwarding. We
+        // should consider adding it in the future to round out the component to
+        // align with input checkbox indeterminate property. Read more here:
+        // https://www.w3schools.com/jsref/prop_checkbox_indeterminate.asp
```

Comment:
**question:** Do you think we could fix this by merging your changes from #1817 (after that PR is landed)? so we could solve this right away :).

Reply:
Yes I like that idea! I landed #1817 but I can't merge the changes until Version Packages are merged, right?

Reply:
good question! Yes, you can merge changes right now :)

You can see it here: https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-core/src/util/add-style.tsx#L48

-----------

For more context, you could say that releasing features are a two-step process: 

1. The "code implementation" changes for a particular PR are merged into `main` right after a PR is landed.
2. The version updates for a particular package are merged into `main` right after `Version Packages` are merged.

This means that, in (1) all the code associated to your PR is directly merged into the main branch as soon as these are landed, and this code is available inside the `wonder-blocks` repo, but it's not available for consumers (like the `webapp` repo) yet. That's where (2) comes into the process, because that's where we make these changes available to consumers by releasing the package changes to npm.

Reply:
Oh I see! Thank you so much for the context. That makes more sense to me now.